diff --git a/config/config.exs b/config/config.exs
new file mode 100644
index 0000000..3447755
--- /dev/null
+++ b/config/config.exs
@@ -0,0 +1,6 @@
+import Config
+
+config :junit_formatter,
+  print_report_file: true,
+  include_filename?: true,
+  include_file_line?: true
diff --git a/mix.exs b/mix.exs
index 2ba5afc..1369248 100644
--- a/mix.exs
+++ b/mix.exs
@@ -23,6 +23,7 @@ defp deps do
     [
       # {:dep_from_hexpm, "~> 0.3.0"},
       # {:dep_from_git, git: "https://github.com/elixir-lang/my_dep.git", tag: "0.1.0"}
+      {:junit_formatter, "~> 3.1", only: [:test]}
     ]
   end
 end
diff --git a/mix.lock b/mix.lock
new file mode 100644
index 0000000..e3e32e9
--- /dev/null
+++ b/mix.lock
@@ -0,0 +1,3 @@
+%{
+  "junit_formatter": {:hex, :junit_formatter, "3.2.0", "6a64d6327c48472e983af7645aeb1d756cad4b33c5ab3a5db820f13d6d2b85fe", [:mix], [], "hexpm", "043d6f660667ee64c6d7e5edb867b7900c395356346e77185879449b50486681"},
+}
diff --git a/test/test_helper.exs b/test/test_helper.exs
index 35fc5bf..d9c0c36 100644
--- a/test/test_helper.exs
+++ b/test/test_helper.exs
@@ -1,2 +1,3 @@
+ExUnit.configure(formatters: [JUnitFormatter, ExUnit.CLIFormatter])
 ExUnit.start()
-ExUnit.configure(exclude: :pending, trace: true)
+# ExUnit.configure(exclude: :pending, trace: true)
